pcr_profile.go

```diff
@@ -169,20 +169,29 @@ func (iter *pcrProtectionProfileIterator) next() pcrProtectionProfileInstr {
 	if len(iter.instrs) == 0 {
```

Comment:
the callers detect this situation implicitly and avoid this panic, but I start to think a more explicit api would make it clearer that they do need to deal with this:

maybe the return of this should be (instr pcrProtectionProfileInstr, exhausted bool) instead of the panic

Reply:
Yeah, I can probably look at this again in a follow up.